MachObfuscator/Mach/ExportTrie/Trie+Parsing.swift

```diff
@@ -8,7 +8,7 @@ extension Trie {
     private var exportedLabels: [[UInt8]] {
         let childrenLabels = children.flatMap { $0.exportedLabels }.map { label + $0 }
         if exportsSymbol {
-            return childrenLabels + [label]
+            return [label] + childrenLabels
```

Comment:
The original results were very counterintuitive while debugging. The new results doesn't matter for runtime.